movies/2.context.md

```diff
@@ -0,0 +1,102 @@
+# The JSON-LD context
+
+## Story
+
+As an API provider, I want to describe the semantics in a JSON-LD context, in order to simplify the actual JSON responses.
+
+## API behaviour
+
+```http
+GET https://hydra-movies.herokuapp.com/
+```
+
+Response body (excerpt):
+
+```json
+{
+  "@context": "/context.jsonld",
+  "@id": "/",
+  "@type": "schema:EntryPoint",
+  "collection": {
+    "@id": "movies",
+    "@type": [
+      "Collection",
+      "mov:Movies"
+    ]
+  }
+}
+```
+
+```http
+GET https://hydra-movies.herokuapp.com/context.jsonld
+```
+
+Response headers (excerpt):
+
+```http
+Last-Modified: Tue, 07 May 2019 08:24:11 GMT
+Etag: W/"a9-16a91649bf8"
+```
+
+Response body (excerpt):
+
+```json
+{
+  "@context": [
+    "https://www.w3.org/ns/hydra/core",
+    {
+      "mov": "https://hydra-movies.herokuapp.com/doc#",
+      "schema": "http://schema.org/"
+    }
+  ]
+}
+```
+
+### Explanation
+
+```
+"@context": "/context.jsonld"
+```
+
+Since the example API uses JSON-LD as serialization format, we need to add a context to the response body to enrich the JSON properties with RDF semantics. A context might get large, so it is a good idea to move it to it's own document and link to it, like shown above.
+
+```
+"@type": "schema:EntryPoint",
+"collection": {
+  "@id": "movies",
+  "@type": [
+    "Collection",
+    "mov:Movies"
+  ]
+}
+```
+
+Given the context, we are able to use simple property names like `collection` within the actual resource representation. Prefixes defined in the context allow short-hand names like `schema:EntryPoint` and `mov:Movies`, that still resolve to fully-qualified URIs when expanded.
```

Comment:
I would move this para after you mention hydra's `@context`. Otherwise the statement about `collection` can be surprising.

Alternatively, I would make it more generic, only mentioning that the context helps remove the URIs from JSON.

And maybe link to https://json-ld.org/spec/latest/json-ld/#the-context somewhere here?

Reply:
I moved it to the bottom. The link you mentioned is already present in the paragraph that is now above that one.